### DISCUSSION TITLE
Quick Question: Is this repo being actively maintained?

### DISCUSSION THREAD

**Discussion by anirudh3LOQ** [NervanaSystems/nlp-architect#233] [General]
Hello All,

Thanks for making this repository available. The project looks really comprehensive and interesting. 

Is it being maintained as the current version is not building or installing due to the dependency on deprecated TF version. 

Any inputs on future plans would help.

Thanks

**Comment by ShaoMinLiu-Holmusk** [comment]
Yeah, apparently some of the dependencies stated in the requirement.txt is self-contradicting, I had to relax some of the specifications to install properly, even after installation, the Bokeh application is a blank page. Does anyone knows how to make it work?
